client/src/components/reader/Resource/Marker.js

```diff
@@ -51,8 +51,8 @@ class ResourceMarker extends Component {
               onMouseLeave={() => { this.setActiveAnnotation(null); }}
             >
               <i
-                className="manicon manicon-cube-fill"
-              ></i>
+                className="manicon resource-cube-fill"
+              >&nbsp;&nbsp;&nbsp;&nbsp;</i>
```

Comment:
We had to add spaces to pad the element/icon without breaking the highlight.  When we applied padding/margin to it, the selection highlight doesn't include the padding.